video/out/d3d11/ra_d3d11.c

```diff
@@ -2386,6 +2386,8 @@ struct ra *ra_d3d11_create(ID3D11Device *dev, struct mp_log *log,
     if (p->fl >= D3D_FEATURE_LEVEL_11_0) {
         ra->caps |= RA_CAP_COMPUTE | RA_CAP_BUF_RW;
         ra->max_shmem = 32 * 1024;
+        // D3D11_CS_THREAD_GROUP_MAX_THREADS_PER_GROUP
+        ra->max_compute_group_threads = 1024;
```

Comment:
Why hard-code this define?

Reply:
I did it because of the max_shmem handling but yeah - just use the constant.